model/minsky.cc

```diff
@@ -295,23 +293,57 @@ namespace minsky
                              }
                          }
                      return false;
-                   });
+                   });               
+    
+    // curved wires not always included in the new group, add them if they aren't. For ticket 1190.
+    if (canvas.selection.numWires()!=g->numWires())
+      {
+        canvas.selection.recursiveDo(&GroupItems::wires,
+                       [&](Wires&, Wires::iterator w) {
+                         auto alreadyAdded = g->findAny
```

Comment:
This is a quadratic algorithm, which is very inefficient. Better to do something with a set, which is O(n log n).